controllers/orders.js

```diff
@@ -0,0 +1,74 @@
+const db = require('../db');
+const PS = require('pg-promise').PreparedStatement;
+
+const psGetMonthlyOrderSummary = new PS({ name: 'monthly-order-summary',
+    text: 'SELECT date_part(\'month\', ordertime) as month, date_part(\'year\', ordertime) as year, count(*) as order_count' +
```

Comment:
I think it's better to use multi line quotes `` here so u don't have to escape the single quotes.

Reply:
Also, i think should use deliveredTime instead if we are counting completed orders. Otherwise, ignore this comment